src/schema-builder/RdbmsSchemaBuilder.ts

```diff
@@ -717,9 +721,13 @@ export class RdbmsSchemaBuilder implements SchemaBuilder {
             // find which columns are new
             const newColumnMetadatas = metadata.columns.filter(
                 (columnMetadata) => {
-                    return !table.columns.find(
-                        (tableColumn) =>
-                            tableColumn.name === columnMetadata.databaseName,
+                    return (
+                        !columnMetadata.isVirtualDecorator &&
+                        !table.columns.find(
```

Comment:
I mean do not put virtual columns into "loadedTables"

Reply:
Are you suggesting a getter method on the columns that filters out the virtual columns (if the table data is loaded into the entity?), if so, I do like that solution better than managing it this was for sure

Reply:
I think the question is: how do we get virtual columns populated into `loadedTables`? It looks likes they aren't, because content of the loadedTables is built from the database columns, not entity metadata columns. If I don't miss anything.

Reply:
Same comment as above:

I have updated this with the changes that you suggested. I was not aware that the table object was being used to be a direct reflection of the table in the database (it makes sense and it cleaned up those checks). Thanks @pleerock !